translations/bnpchs-2018-penyisihan-lampar-dadu.html

```diff
@@ -0,0 +1,50 @@
+<!--Tossing dice-->
+
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Jelix has two dices each with $$$N$$$ sides, he has an idea of a game with those two dices. On each side of the dice, there is a number between $$$1$$$ and $$$N$$$ inclusive and the numbers on each side of a dice is different. This game is played by 2 players, player one picks a number between $$$1$$$ and ($$$N - 1)$$$ inclusive. Then, both players toss their dice, if the maximum number that appears on the top of both dice is smaller or equal to the number picked by the first player, then the first player wins the game. If not, then the second player wins.
+</p>
+
+<p style="text-align:justify">
+  After making the rules, Jelix feels like the first player will have a better chance of winning. So, Jelix will always play as the first player.
+</p>
+
+<p style="text-align:justify">
+  This time Jelix challenges Fing to play with him. Because Jelix and Fing are best friends, Jelix does not want Fing to feel suspicious by picking a number that is too large and making Fing's winning chance very low. Jelix decided to pick a number in a way that Jelix's winning rate is higher than Fing's but the difference between their winning rate must be as small as possible. For example, Jelix's winning rate is 50.05% and Fing's winning rate is 49.95%. Help Jelix decide which number he should pick.
```

Comment:
```suggestion
  This time Jelix challenges Fing to play with him. Because Jelix and Fing are best friends, Jelix does not want Fing to feel cheated by picking a number that is too large and resulting in Fing's winning chance being very low. Jelix decided to pick a number such that Jelix's winning rate is higher than Fing's but the difference between their winning rate must be as small as possible. For example, Jelix's winning rate is 50.05% and Fing's winning rate is 49.95%. Help Jelix decide which number he should pick.
```